launcher/image/test/test_cloud_init_userdata_disabled.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
suggestion (can be in a follow up): this test and other "negative tests" should be written so that they can run on both debug and hardened. that way, we can see these tests pass in one environment and fail in the other.

Reply:
speaking of, were you able to see this script fails on a non-hardened image?

Reply:
I think this test can run on debug image.
But this specific test should also pass on debug image, because user-data cloudinit is also disabled on debug image.

We can try to run this test on a regular cos image, and then it should fail.